CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Fixed`
 
+- [#136](https://github.com/nf-core/epitopeprediction/pull/135) - Allow variants with unique annotation fields
```

Comment:
I think it should be #135 at the beginning and you could change the description to something like `Fix unique variant annotation field handling  #136` e.g.